aiida/tools/archive/imports.py

```diff
@@ -117,6 +129,7 @@ def import_archive(
     type_check(test_run, bool)
     backend = backend or get_manager().get_profile_storage()
     type_check(backend, StorageBackend)
+    qparams = QueryParams(batch_size=batch_size, filter_size=filter_size)
```

Comment:
nitpick: why not just have `query_params` for increased legibility. The few extra characters shouldn't hurt, should they?